### ISSUE TITLE
zfs-utils fails to build

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#427]
It fails with this error:

```
==> Missing mandatory directory: /usr/lib/modules. Exiting!
error: command failed to execute correctly
umount: /winespaos/dev: target is busy.
```

This leads to the rest of the `winesapos-install.sh` script failing because `arch-chroot` does not work after this.

The reason this error happens is because the Linux kernels are installed after ZFS. We need to install the kernels first.

**Comment by LukeShortCloud** [comment]
Installing the Linux kernels first gets building the `zfs-dkms` slightly further but it still fails leading to `udev` mount issues.

```
/usr/bin/dkms: line 1033: sha512: command not found
==> depmond 5.15.52-1-lts
umount: /winesapos/dev: target is busy.
```

**Comment by LukeShortCloud** [comment]
This latest error is actually an upstream bug in the DKMS project that broke functionality on Arch Linux:

- https://bbs.archlinux.org/viewtopic.php?id=277700
- https://github.com/dell/dkms/issues/229

**Comment by LukeShortCloud** [comment]
The first error `==> Missing mandatory directory: /usr/lib/modules. Exiting!` has been solved via: https://github.com/LukeShortCloud/winesapOS/commit/741e137dbd7f756d75daab2d2f3f1a009a455bda

The second error `/usr/bin/dkms: line 1033: sha512: command not found` is an upstream issue. I will keep this GitHub issue open to monitor the status of it being fixed in Arch Linux. I did a build of Manjaro and this was not an issue on there.
